src/components/plugin-button/plugin-button.tsx

```diff
@@ -8,18 +8,16 @@ const {Tooltip, Icon} = KalturaPlayer.ui.components;
 
 interface PluginButtonProps {
   isActive: boolean;
-  onClick: OnClick;
   setRef?: (ref: HTMLButtonElement | null) => void;
   id: string;
   icon: string;
   label?: string;
   dataTestId?: string;
 }
 
-export const PluginButton = ({isActive, onClick, label, id, icon, dataTestId, setRef}: PluginButtonProps) => {
+export const PluginButton = ({isActive, label, id, icon, dataTestId, setRef}: PluginButtonProps) => {
   return (
     <Tooltip label={label} type="bottom">
-      <A11yWrapper onClick={onClick}>
```

Comment:
1) check a11y by open plugin by keyboard (space, enter);
2) fix focus move on navigation search if plugin opened by keyboard (Cypress tests caught regression after the fix):